lib/tasks/csv.rake

```diff
@@ -0,0 +1,37 @@
+desc "Scrape companies"
+
+task :make_companies => :environment do 
+  require 'open-uri'
+  require 'csv'
+
+  file_path = "lib/tasks/district_eleven.csv"
```

Comment:
I would suggest to call you files `district_1.csv`, `district_2.csv` etc. so that you can much more easily create a loop that will scrape all files